spec/unit/room.spec.ts

```diff
@@ -3562,6 +3562,11 @@ describe("Room", function () {
                 lastEventInMainTimeline = addRoomMainAndThreadMessages(room, 42, 23).mainEvent!;
                 expect(room.getLastLiveEvent()).toBe(lastEventInMainTimeline);
             });
+
+            it("and both events have the same timestamp, it should return the last event from the main timeline", () => {
```

Comment:
Should it? I'm not sure what bug that would fix. 
How does the main timeline take precedence? If two events have the same `ts` I don't think we have an accurate way of knowing in our current setup their order in the DAG hosted by synapse

Reply:
I will revert this change for this PR. We need to fix the type error.